pykafka/simpleconsumer.py

```diff
@@ -393,8 +394,17 @@ def commit_offsets(self):
                 log.debug("Retrying")
             time.sleep(i * (self._offsets_channel_backoff_ms / 1000))
 
-            response = self._offset_manager.commit_consumer_group_offsets(
-                self._consumer_group, 1, b'pykafka', reqs)
+            try:
+                response = self._offset_manager.commit_consumer_group_offsets(
+                    self._consumer_group, 1, b'pykafka', reqs)
+            except (SocketDisconnectedError, IOError):
+                log.error("Error committing offsets for topic %s "
+                          "(SocketDisconnectedError)",
+                          self._topic.name)
+                if i >= self._offsets_commit_max_retries - 1:
+                    raise
+                continue
```

Comment:
Cool, thanks for this. Should this call `self._update()` before `continue` here?